lib/bibliothecary.rb

```diff
@@ -10,12 +10,17 @@
 
 module Bibliothecary
   def self.analyse(path)
+    file_list = load_file_list(path)
+    package_managers.map{|pm| pm.analyse(path, file_list) }.flatten.compact
+  end
+
+  def self.load_file_list(path)
```

Comment:
This refactor seems fine but is unrelated? Should we now do better testing of `load_file_test`?

Reply:
I split that out to enable better testing of it but I didn't write the tests yet. I can include them with this PR if that's better.